packages/expo-image/README.md

```diff
@@ -21,34 +21,44 @@ A cross-platform, performant image component for React Native and Expo.
 
 ## Supported image formats
 
-|   Format   | Android | iOS |                        Web                        |
-| :--------: | :-----: | :-: | :-----------------------------------------------: |
-|    WebP    |   ✅    | ✅  |   ✅ [~96% adoption](https://caniuse.com/webp)    |
-| PNG / APNG |   ✅    | ✅  | ✅ / ✅ [~96% adoption](https://caniuse.com/apng) |
-|    AVIF    |   ✅    | ✅  |   ⏳ [~79% adoption](https://caniuse.com/avif)    |
-|    HEIC    |   ✅    | ✅  |  ❌ [not adopted yet](https://caniuse.com/heif)   |
-|    JPEG    |   ✅    | ✅  |                        ✅                         |
-|    GIF     |   ✅    | ✅  |                        ✅                         |
-|    SVG     |   ✅    | ✅  |                        ✅                         |
-|    ICO     |   ✅    | ✅  |                        ✅                         |
-|    ICNS    |   ❌    | ✅  |                        ❌                         |
+|   Format   | Android | iOS |                      Web                       |
+| :--------: | :-----: | :-: | :--------------------------------------------: |
+|    WebP    |   ✅    | ✅  |                       ✅                       |
+| PNG / APNG |   ✅    | ✅  |                       ✅                       |
+|    AVIF    |   ✅    | ✅  |  ⏳ [~87% adoption](https://caniuse.com/avif)  |
+|    HEIC    |   ✅    | ✅  | ❌ [not adopted yet](https://caniuse.com/heif) |
+|    JPEG    |   ✅    | ✅  |                       ✅                       |
+|    GIF     |   ✅    | ✅  |                       ✅                       |
+|    SVG     |   ✅    | ✅  |                       ✅                       |
+|    ICO     |   ✅    | ✅  |                       ✅                       |
+|    ICNS    |   ❌    | ✅  |                       ❌                       |
 
 # API documentation
 
-- [Documentation for the latest release](https://docs.expo.dev/versions/unversioned/sdk/image/)
+- [Documentation for the latest stable release](https://docs.expo.dev/versions/latest/sdk/image/)
 
-# Installation
+# Installation in managed Expo projects
 
-> Currently `expo-image` can be used only with [development builds](https://docs.expo.dev/develop/development-builds/introduction/), in Expo Go, and bare React Native apps with [configured Expo modules](https://docs.expo.dev/bare/installing-expo-modules/).
-> It is not available with Snack yet.
+For [managed](https://docs.expo.dev/archive/managed-vs-bare/) Expo projects, please follow the installation instructions in the [API documentation for the latest stable release](https://docs.expo.dev/versions/latest/sdk/image/).
```

Comment:
```suggestion
For [managed](https://docs.expo.dev/archive/managed-vs-bare/) Expo projects, follow the installation instructions in the [API documentation for the latest stable release](https://docs.expo.dev/versions/latest/sdk/image/).
```

One small suggestion to avoid words like "please".

Reply:
This is what we have in the template: https://github.com/expo/expo/blob/d42674624cf4d034d01fde4da0a44d0818da9ca7/packages/expo-module-scripts/templates/README.md?plain=1#L14
And I guess also in many other modules.